completions/_modules

```diff
@@ -21,30 +21,50 @@
 # being sourced before it and thus before the `module' alias has been defined.
 [[ -f /etc/profile.d/modules.sh ]] || return 1
 
-# TODO:API: rename per conventions, rework to use vars rather than outputting
+# @deprecated 2.12 Use `_comp_cmd_module__compgen_list`
```

Comment:
The replacements of these are functions private to `_modules`, so I think we can just remove the old versions.

Reply:
Removed these old names.